ts/src/config.ts

```diff
@@ -144,5 +162,9 @@ export const defaultConfig = {
   // This is the largest duration for setTimeout which does not cause it to
   // run immediately.
   // https://nodejs.org/dist/latest-v9.x/docs/api/timers.html#timers_settimeout_callback_delay_args.
-  serverBackoffCapMillis: 2147483647
+  serverBackoffCapMillis: 2147483647,
+
+  localProfilingPeriodMillis: 1000,
+  localLogPeriodMillis: 1000,
```

Comment:
IIRC, this was 10000 in the code. Is it intentionally 1000 now?

Reply:
Fixed.